src/decorator_trait.jl

```diff
@@ -330,20 +330,24 @@ function is_point(
     kwargs...,
 )
     # to be safe check_size first
+    try
+        pt = is_point(get_embedding(M, p), embed(M, p), te; kwargs...)
+        !pt && return false # no error thrown (deactivated) but returned false -> return false
+    catch e
+        if e isa DomainError || e isa AbstractManifoldDomainError
+            e = ManifoldDomainError(
+                "$p is not a point on $M because it is not a valid point in its embedding.",
+                e,
+            )
+            te && throw(e)
```

Comment:
```suggestion
            ew = ManifoldDomainError(
                "$p is not a point on $M because it is not a valid point in its embedding.",
                e,
            )
            te && throw(ew)
```
Maybe let's call the new variable a different name to avoid confusion.

Reply:
sure, I started with the throw outside the if, then it was easier to keep the variable name.